index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=poppins">
+    <link rel="stylesheet" href="./css/style.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <a class="logo" href="https://www.linkedin.com/in/jalal-ahmadi-58312b270/"><b>My Lgo</b></a>
+        <a href="#"><i class="material-icons">	list</i></a>
+    </header>
+    <div class="first-container">
+    <section class="container">
+    
+        <div class="content">
+            <h1><b>I’m Alexandra.<br>
+
+                Glad to see<br>
+                you!</b></h1>
+            <p>I’m a software developer! I can help youbuild a product , feature or website Lookthrough some of my work and
+                experience! If you like what you see and
+                have a project you need coded, don’t
+                hesitate to contact me.<br>
+                <span>LETS CONNECT</span></p>
+        <ul class="icons">
+            <li><a class="icon1" href="https://twitter.com/JalalAh78951382"><img class="icon1" src="Vector (1).jpg" alt=""></a></li>
+            <li><a class="icon2" href="https://www.linkedin.com/in/jalal-ahmadi-58312b270/"><img class="icon2" src="Vector (2).jpg" alt=""></a></li>
+            <li><a class="icon3" href="https://www.linkedin.com/in/jalal-ahmadi-58312b270/"><img class="icon3" src="Vector (3).jpg" alt=""></a></li>
+            <li><a class="icon4" href="https://www.linkedin.com/in/jalal-ahmadi-58312b270/"><img class="icon4" src="Vector (4).jpg" alt=""></a></li>
+            <li><a class="icon5" href="#"><img class="icon5" src="Vector (5).jpg" alt=""></a></li>
+            <li><a class="icon6" href="https://github.com/jalal1379"><img class="icon6" src="Vector (6).jpg" alt=""></a></li>
+            <li><a class="icon7" href="#"><img class="icon7" src="Vector (7).jpg" alt=""></a></li>
+        </ul>
+            
+        </div>
+    
+        <div class="cards">
+            <div class="card1">
+            <img src="Snapshoot Portfolio (5).jpg" alt="">
+            <h1>Tonic</h1>
+            <h2>CANOPY</h2>
+            <ul class="date">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately
+                personalized reads; no accounts orsign-ups required.</p>
+                <ul class="language">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button>See project</button>
+                
+            <div class="card2">
```

Comment:
Great job on the card! However, there are a few improvements that can be made to align with the design and improve the code structure:

- [ ] Firstly, the other cards should be separate entities instead of being contained within a single card. Make sure to review the card structure and separate the individual cards accordingly.
- [ ] Secondly, it's recommended to use the `<article>` tag instead of `<div>` for the card element. This will ensure better semantic structure and enhance accessibility.
- [ ] Additionally, consider using CSS Grid to align the items within the work section and utilize flexbox properties inside the card elements. This will provide more flexibility in positioning and layout management.
- [ ] Please consider font styles to make your code close to the design.

| Your | Designe|
| --- | --- |
| ![image](https://github.com/jalal1379/Microverse_portflaio/assets/55840999/74b3a4fd-50d6-4e9f-b94f-9cbee7c3ecde) | ![image](https://github.com/jalal1379/Microverse_portflaio/assets/55840999/5eca18fa-7325-44a0-8473-a500200f2f3e) | 


By implementing these changes, you will enhance the structure, semantics, and visual alignment of the cards in accordance with the design. Keep up the great work! 👍